database-seed/mongo-loader.js

```diff
@@ -28,7 +18,8 @@ const seedDatabase = () => {
         });
         Promise.all(updates).then(() => {
             console.log('Database seeded');
-            process.exit();
+            mongoose.connection.close();
```

Comment:
What's the difference between process.exit and connection.close?

Reply:
`process` refers to the global object in a `node` environment.  Using `process.exit()` kills the entire node runtime, which was a hacky way to solve the problem I was encountering.